docs/tutorial/part-two/index.md

```diff
@@ -298,36 +298,36 @@ export default () => (
 )
 ```
 
-> Tip: Generally, if you use a component in multiple places on a site, it should be in its own module file in the `components` directory. But, if it's used only in one file, create it inline.
+> Wskazówka: Generalnie, jeśli użyjesz komponentu w kilku różnych miejscach na stronie, powinien on znajdować się we własnym pliku-module w folderze 'components'. Ale gdy jest użyty tylko w jednym pliku, napisz go bezpośrednio w kodzie.
```

Comment:
```suggestion
> Wskazówka: Generalnie, jeśli zamierzasz używać komponentu w kilku różnych miejscach na stronie, powinien znajdować się on we własnym Module CSS w folderze `components`. Ale jeśli jest użyty tylko w jednym pliku, umieść go bezpośrednio w kodzie.
```

Reply:
Mysle ze ostatnie zdanie powinno zawierać sugestie że jest to opcjonalne... Czasami piszemy przecież komponenty na pojedynczy użytek w odrebnych plikach, bo nie wiadomo kiedy się przyda. Także nie chciałbym wprowadzić kogoś w błąd gdyby omyłkowo zrozumiał że jak używasz komponentu raz, to zawsze ładuj go bezpośrednio do pliku w którym zostanie użyty

Reply:
myślisz, że zamiana na 'możesz umieścić go bezpośrednio w kodzie' wystarczy, czy potrzebujemy czegoś bardziej dosadnego?

Reply:
myślę że słówko 'możesz' załatwia sprawę!

Reply:
Tutaj chyba nie zmieniliście tego co? Bo tylko tego nam brakuje do merge z tego co widzę

Reply:
outdated, dałem nową sugestie: https://github.com/gatsbyjs/gatsby-pl/pull/7/files#r341823142